Makefile

```diff
@@ -68,6 +68,10 @@ generate-internal:
 tidy: ## Run go mod tidy
 	go mod tidy -v
 	cd hack; go mod tidy -v
```

Comment:
I would use && to separate these commands rather than ;. If the first command fails when you use && then the second command won't run.